README.md

```diff
@@ -95,33 +95,63 @@ also be asked to sign a
 
 ## Development Internals
 
-### Building from source
+### Installing Poetry
 
 Rasa uses Poetry for packaging and dependency management. If you want to build it from source,
 you have to install Poetry first. This is how it can be done:
 
-```
-curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3
+```bash
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
 ```
 
 There are several other ways to install Poetry. Please, follow 
 [the official guide](https://python-poetry.org/docs/#installation) to see all possible options.
 
-To install dependencies and `rasa` itself in editable mode execute
+### Managing environments
+
+The official guide suggests to use [pyenv](https://github.com/pyenv/pyenv) or any other similar tool
+to easily switch between Python versions. This is how it can be done:
+
+```bash
+pyenv install 3.7.6
+pyenv local 3.7.6  # Activate Python 3.7.6 for the current project
+```
```

Comment:
does it really make sense to bring up `pyenv` here? Most people will just have one Python version installed anyway